src/components/GlobalNav/RoleSwitcher/RoleSwitcher.jsx

```diff
@@ -34,7 +35,7 @@ export const RoleSwitcher = ({ users, login, createUser }) => (
     ))}
     <RoleItem roleAction={createUser} />
   </IconMenu>
-  : <span>No Demo Session</span>
+  : <div><LoadingSpinner color={palette.primary3Color}/></div>
```

Comment:
Did this so the spinner can margin center align properly. Probably a better way to do it in the spinner component itself